.bithoundrc

```diff
@@ -42,17 +42,13 @@
     "**/extjs/src/**",
     "**/extjs/welcome/**",
     "bower_components/**",
-    "**/assets/**"
+    "**/assets/**",
+    "public/js/tracking/**"
```

Comment:
Should this be a bit more wide ranging? Given that the dir will have generated assets in? Or maybe wait until that time i.e. when the assets are being generated?